index.html

```diff
@@ -69,5 +77,296 @@
       </ul>
       <div class="some"></div>
     </div>
+    <div class="recent-works">

```

Comment:
- [ ] Kindly make this a `section` tag instead of a `div` tag. It is more semantic to use a `section` tag for standalone sections on a webpage. 👍